client/components/inventory.jsx

```diff
@@ -13,33 +17,70 @@ export default class Inventory extends React.Component {
       .catch(err => console.error(err));
   }
 
-  render() {
-    const articles = this.state.articles;
-    const numPlaceholders = [];
-    if (articles.length % 2 === 0) {
-      for (let i = 0; i < 8; i++) {
-        numPlaceholders.push('placeholder');
+  handleTypeSelect(event) {
+    const articleType = event.target.value;
+    if (articleType === 'articles') {
+      fetch('/api/inventory/1')
+        .then(res => res.json())
+        .then(articles => this.setState({
+          articles,
+          articleType
+        }))
+        .catch(err => console.error(err));
+      return;
+    }
+    fetch(`/api/inventory/1/${articleType}`)
+      .then(res => res.json())
+      .then(articles => this.setState({
+        articles,
+        articleType
+      }))
+      .catch(err => console.error(err));
```

Comment:
Since these fetches are almost identical, you should be able to use your conditional or a ternary expression to just choose the url.